.github/workflows/validate.yml

```diff
@@ -0,0 +1,123 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: Auto Validate Examples
+
+on:
+  workflow_dispatch:
```

Comment:
For now manual trigger. Can change later.

Reply:
I have it set to run on every push for python-sdk, because why not?

Reply:
Add to pull requests and pushes. This is not different from our ITs.

Reply:
Let me fix that.

Reply:
Done.